packages/compass-components/src/components/interactive-popover.spec.tsx

```diff
@@ -17,9 +17,9 @@ function renderPopover(
       hideCloseButton={props?.hideCloseButton}
       customFocusTrapFallback={`#${innerContentTestId}`}
       setOpen={() => {}}
-      trigger={({ onClick, ref, children }) => (
+      trigger={({ onClick, children }) => (
```

Comment:
This doesn't look right, ref needs to be passed to the trigger for the component to work (looking at the test, we just don't test the behavior that relies on this)